packages/wallet/frontend/src/lib/api/paymentPointer.ts

```diff
@@ -74,6 +74,14 @@ type BasePaymentPointerArgs = {
   paymentPointerId: string
 }
 
+type PaymentPointeOP = {
+  id: string
+  assetCode: string
+  assetScale: number
+  publicName: string
+  authServer: string
+}
+
```

Comment:
Add an `AssetOP` in `lib/api/asset.ts` and import it from there:
```suggestion
type PaymentPointerOP = AssetOP & {
  id: string
  publicName: string
  authServer: string
}

```